doc/content/examples/example3.md

```diff
@@ -87,4 +87,36 @@ DEEPSOIL and presented in
 Further details regarding to application of MASTODON to simulate seismic site response and soil
 structure interaction analysis can be found in [!citet](baltaji2017nonlinear).
 
+## Implementation using Automatic Differentiation
+
+This example can also be run using Automatic Differentiation (AD). Automatic Differentiation
+computes the numerical Jacobian that is very close to the theoretical Jacobian.  
+More information can be found in [ISoil material](source/materials/ADComputeISoilStress.md).
+
+The following two additions to the input file should be made to run this example using AD.
```

Comment:
Please change 'additions' to 'modifications' and add the following text:

The following additions to the input file should be made to run this example using AD. These modifications ensure the the AD versions of the stress divergence kernels and the material kernels are being used during the solve.